Makefile

```diff
@@ -0,0 +1,15 @@
+PROJECT_NAME = substreams-sink-sql
+DATE := $(shell date '+%Y-%m-%dT%H:%M:%S')
+HEAD = $(shell git rev-parse HEAD)
+LD_FLAGS = -X main.version=$(shell git describe --tags --always --dirty) -X main.commit=$(shell git rev-parse HEAD) -X main.date=$(DATE)
```

Comment:
`main.commit` and `main.date` has no effect here, we do not define those variables in `main` package.